src/guides/v2.4/release-notes/b2b-release-notes.md

```diff
@@ -2,3 +2,40 @@
 group: release-notes
 title: B2B Release Notes
 ---
+
+The release notes for the B2B extension captures additions and bug fixes that Magento has added during a release cycle.
+These release notes can include:
+
+-  {:.new}New features
+-  {:.fix}Fixes and improvements
+
+## Magento B2B - Version 2.4.0
+
+-  {:.new} Purchase Orders have been enhanced and rewritten. The are now included my default in Magento Commerce.
+-  {:.new} Purchase Order Approval Rules have been implemented. This allows users to control the Purchase Order workflow by creating purchasing rules for orders.
+-  {:.new} Login as Customer is now included by default in Magento Commerce. This allows site employees to assist customers by logging in as the customer to see what they see.
+
+-  {:.fix} Attribute aggregations are now working correctly for Layered Navigation with Elasticsearch
+-  {:.fix} Search orders by special chars is now working properly.
+-  :.fix} Clicking on &quot;Clear All&quot; Button now expands all the filters, rather than collapsing them.
+-  {:.fix} Product SKU/Name is now included in the Order History search filter summary.
+-  {:.fix} Sorting Indicator is now displayed properly in the My Purchase Orders Grid.
+-  {:.fix} 'https://jira.corp.magento.com/browse/B2B-564'>B2B-564 - Now, you can click the Approve/Cancel/Reject/Purchase Purchase Order button only once. Previously, you could click the button multiple times.
+-  {:.fix} The Purchase Order Approve, Reject, Cancel, Validate buttons now render correctly on mobile devices.
```

Comment:
In the previous item, the buttons were separated by slashes. If you change the separator to commas, add "and" before the last button,